.travis.yml

```diff
@@ -9,14 +9,27 @@ cache:
   directories:
     - node_modules
 
+env:
+    - EMBER_TRY_SCENARIO=default
+    - EMBER_TRY_SCENARIO=ember-release
+    - EMBER_TRY_SCENARIO=ember-beta
+    - EMBER_TRY_SCENARIO=ember-canary
+
+matrix:
+    fast_finish: true
+    allow_failures:
+        - env: EMBER_TRY_SCENARIO=ember-beta
+        - env: EMBER_TRY_SCENARIO=ember-canary
+
 before_install:
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+    - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
```

Comment:
Remove this line.

Travis CI rolled back the commit that was associated with this configuration.  It is possible to have Travis CI install our own copy of PhantomJS 2.0 (see https://github.com/ember-cli/ember-cli/pull/3895#issuecomment-103736634) but it's not as easy to do so on a development machine to be in sync.  There is not a yet a 2.0 npm version of PhantomJS so would need to pull the PhantomJS package down ourselves, though there are bugs in the package (see http://phantomjs.org/download.html) and need to build from source so it is a lot of effort right now to use v2.x versus just sticking with 1.9.8.  Unfortunately that means we are also stuck at 1.9.8 and not the latest (1.9.17) since Travis CI is locked into this version.